src/Api/Controllers/PlansController.cs

```diff
@@ -25,15 +25,6 @@ public ListResponseModel<PlanResponseModel> Get()
         return new ListResponseModel<PlanResponseModel>(responses);
     }
 
-    [HttpGet("all")]
-    [AllowAnonymous]
-    public ListResponseModel<PlanResponseModel> GetAllPlans()
-    {
-        var data = StaticStore.Plans;
-        var responses = data.Select(plan => new PlanResponseModel(plan));
-        return new ListResponseModel<PlanResponseModel>(responses);
-    }
```

Comment:
Note to self: still need to review changes in clients and make sure it covers this

Reply:
Update: yes, clients PR has been updated accordingly